README.md

```diff
@@ -0,0 +1,40 @@
+### Documentation is included in the Documentation folder ###
```

Comment:
Remove all screenshots from .screenshots and add the folder to the .gitignore file

Reply:
Added screenshots folder to gitignore and removed all screenshots